data/ui/preferences.blp

```diff
@@ -12,13 +12,15 @@ template $GraphsPreferencesWindow : Adw.PreferencesWindow {
 
       Adw.ComboRow center {
         title: _("Center Action Behaviour");
+        subtitle: _("Which value to center on when performing a center action");
         model: StringList{
-          strings [_("Center at maximum Y value"), _("Center at middle coordinate")]
+          strings [_("Center at maximum Y value"), _("Center at middle X value")]
```

Comment:
The main reason for this is that "Center at middle coordinate" is just long enough that it cuts off the last part of the sentence, usign "X value" instead is consistent with the other option and is a bit shorter. But I think you are correct that this logic should applied in the gschema and `migration.py` as well. Should be a bit careful of adding too much overhead to the `migration.py` and not change strings when not needed, so I'm open to keeping this string as-is for now and re-evaluate all strings later in a seperate PR before next release.

In this specific case, I just checked, and it seems to work fine without any new logic in `migration.py`. When I set it to "center at max Y" in Graphs 1.6, it is still at "center at max Y" in this commit. And when I set it to "center at middle X" in graphs 1.6, it is still at "center at middle X" in this latest commit.

This could be because "enum 0" (center at max Y) is unchanged in string, so that setting will be nicely copied over. And "enum 1" (center at middle X) is the default. So when reverting to this default value with this changing key, it still ends up correctly. I maybe need to think about it to be sure, but I think in this specific case new migration logic would not be necessary.